### PR TITLE
Update Editors, Authors, contributors and acknowledgements.

### LINKED ISSUE #675: Update list of Editors/Authors for spec.
I ran a few tools to determine who has contributed to the specification and in what ways. We need to be careful with these numbers because they don't tell the entire story. For example, there is a ton of work that goes into initial design, offline discussions, implementations, etc. It's also dangerous to count quantity over quality. So, take these numbers with a grain of salt.

The first statistic is lines of code, commits, and file changes. I used a tool called git-fame to do the calculation. The sorting matches GitHubs algorithm pretty closely. The sorted order is below for the top contributors:

```
msporny@tao:~/work/vc-data-model$ git-fame -w -C -M .

| Author               |   loc |   coms |   fils |  distribution   |
|:---------------------|------:|-------:|-------:|:----------------|
| Manu Sporny          |  2430 |    479 |     67 | 66.2/49.3/27.3  |
| Grant Noble          |   240 |     76 |     11 | 3.8/ 4.5/ 2.4   |
| Dave Longley         |   117 |     30 |      7 | 3.2/ 3.1/ 2.9   |
| Dan Burnett          |   116 |     69 |     13 | 3.2/ 7.1/ 5.3   |
| Brent Zundel         |   111 |     43 |     13 | 3.0/ 4.4/ 5.3   |
| Gregg Kellogg        |    86 |     26 |     15 | 2.3/ 2.7/ 6.1   |
| David-Chadwick       |    78 |     31 |     17 | 2.1/ 2.6/ 6.1   |
| Ted Thibodeau Jr     |    78 |     35 |      3 | 2.1/ 3.6/ 1.2   |
| Oliver Terbu         |    73 |      9 |      4 | 2.0/ 0.9/ 1.6   |
| David I. Lehn        |    43 |     18 |      7 | 1.2/ 1.9/ 2.9   |
| Matthew Collier      |    37 |      1 |      5 | 1.0/ 0.1/ 2.0   |
```

I also wrote a tool to pull every single comment from the specification repository and tally the results. The score is the number of comments, plus the number of bytes written divided by 1024. Yes, that's arbitrary, but the arbitrary rule seemed somewhat fair in calculating a final score as it was applied equally across every contributor. Again, the top rankings are shown below:

```
                user |  score |   comments |    bytes |
---------------------------------------
             msporny |   1237 |        810 |   437940 |
      David-Chadwick |    694 |        445 |   255881 |
            dlongley |    366 |        205 |   165495 |
            jandrieu |    305 |        137 |   172485 |
         brentzundel |    263 |        186 |    79741 |
           stonematt |    227 |        168 |    61143 |
            burnburn |    204 |        170 |    34825 |
             nadalin |    167 |        141 |    26942 |
             TallTed |    126 |         72 |    56063 |
            agropper |    108 |         33 |    77424 |
              Drabiv |     94 |         49 |    46694 |
              RieksJ |     75 |         39 |    37402 |
        kimdhamilton |     55 |         30 |    25631 |
        ChristopherA |     48 |         32 |    16398 |
               awoie |     44 |         33 |    12217 |
              chaals |     41 |         31 |    10867 |
            gkellogg |     34 |         25 |     9538 |
          grantnoble |     31 |         29 |     2972 |
```

My suggestion is that we use these numbers to guide who is listed as an Editor and as an Author. Anyone remaining will be specifically called out and thanked above and beyond the typical "Acknowledgements" section.

### PR DESCRIPTION
**Pull request by msporny**
Fixes #675.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/676.html" title="Last updated on Jul 3, 2019, 1:54 AM UTC (ab9a532)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/676/cd28fa0...ab9a532.html" title="Last updated on Jul 3, 2019, 1:54 AM UTC (ab9a532)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -125,6 +127,33 @@
...
+Joe Andrieu, David I. Lehn, Matthew Collier, and Adrian Gropper.
+      </p>
+
+      <p>
+Work on this specification has been supported by the Rebooting the
+Web of Trust community facilitated by Christopher Allen, Shannon Appelcline,
+Kiara Robles, Brian Weller, Betty Dhamers, Kaliya Young, Manu Sporny,
+Drummond Reed, Joe Andrieu, Heather Vescent, Samantha Chase, and Andrew Hughes.
+The participants in the Internet Identity Workshop facilitated by 
+Phil Windley, Kaliya Young, Doc Searls, and Heidi Nobantu Saul also
```
Comment (TallTed) [inline index.html]:
```
Workshop facilitated by 
Phil Windley, Kaliya Young, Doc Searls, and Heidi Nobantu Saul also
```
->
```
Workshop, facilitated by 
Phil Windley, Kaliya Young, Doc Searls, and Heidi Nobantu Saul, also
```
or
```
Workshop (facilitated by 
Phil Windley, Kaliya Young, Doc Searls, and Heidi Nobantu Saul) also
```

File: index.html [hunk 2]
```diff
@@ -125,6 +127,33 @@
...
+
+      <p>
+Work on this specification has been supported by the Rebooting the
+Web of Trust community facilitated by Christopher Allen, Shannon Appelcline,
+Kiara Robles, Brian Weller, Betty Dhamers, Kaliya Young, Manu Sporny,
+Drummond Reed, Joe Andrieu, Heather Vescent, Samantha Chase, and Andrew Hughes.
+The participants in the Internet Identity Workshop facilitated by 
+Phil Windley, Kaliya Young, Doc Searls, and Heidi Nobantu Saul also 
+supported the refinement of this work through numerous working sessions 
+designed to educate, debate, and improve the content in this specification.
```
Comment (TallTed) [inline index.html]: `designed to educate, debate, and improve the content in this specification` needs some rework, as it currently reads to me as a shortening of `designed to educate the content, debate the content, and improve the content in this specification`.

Maybe `designed to educate about, debate on, and improve the content of this specification`, as these verb forms can all reasonably apply to either "this specification" or "the content of"?

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html]: Fixed in ab9a5322046d152943b2ff5e4f51002e1fc76760

File: index.html (hunk 2 above)
Comment (msporny) [reply index.html]: Fixed in ab9a5322046d152943b2ff5e4f51002e1fc76760.

**Comment by msporny** [comment]
Multiple positive reviews, editorial, merging.

**Pull request merged**